### ISSUE TITLE
Errors with ExtractTextPlugin and MiniCssExtractPlugin

### ISSUE THREAD

**Issue by bee-po** [agoldis/webpack-require-from#9]
Hello,

We have 2 projects, the first one is using webpack 3 with ExtractTextPlugin and the second is using webpack 4 with MiniCssExtractPlugin.

In both of these projects, we have a lot of errors like this one :

```
Error: WebpackRequireFrom: 'getChunkURL' is not a function or not available at runtime. See https://github.com/agoldis/webpack-require-from#troubleshooting
    at Function.get [as p] (H:\project\node_modules\css-loader\index.js!H:\project\node_modules\less-loader\dist\cjs.js??ref--10-2!H:\project\no
de_modules\dep\example.less:67:21)
```

Hopefully, everything is working but the log is now quite annoying to read...
It seems that the webpack-require-from plugin is going through our CSS chunks. Is it possible to ignore these chunks ? If not, can you consider the possibility to add an option to filter chunks ?

Thanks!

**Comment by agoldis** [comment]
Hey! Thanks for opening the issue! Would you mind please sharing your webpack configuration and if possible, code that reproduces the problem.

I will look into

**Comment by agoldis** [comment]
@bee-po hey, I think I can implement some kind of filtering, it will require some work, meanwhile I can suggest to:
1. Make sure `getChunkURL` is available before first chunk loading attempt
2. I will try to implement errors suppression quickly to prevent showing errors

**Comment by bee-po** [comment]
Hi, thank you very much for your quick answer!

here is an example of code reproducing the problem: https://github.com/bee-po/webpack-require-from-issue-9

**Comment by agoldis** [comment]
@bee-po thank for creating the repo! it is indeed very useful, I am still exploring how to resolve the issue properly, meanwhile you can use `supressErrors` option to mute the errors during compilation time as a workaround. It is available in version `1.3.0`.

**Comment by agoldis** [comment]
Seems like suppressing errors logging is good enough for now. 
Closing, please reopen if needed.
